### ISSUE TITLE
Pre-training of Expression task -  loss always 0

### ISSUE THREAD

**Issue by amruz** [cambridge-mlg/weighted-retraining#3]
While pre-training the model for expression task the loss is always 0 , both train and validation. Any ideas what went wrong? I used the same version of tensorflow and keras as mentioned in the github.
Thanks in advance

**Comment by AustinT** [comment]
Very sorry that we have not responded to this earlier. I don't have an idea of what went wrong; you didn't provide a lot of detail. I will close this issue since it has been so long, but if you are still experiencing this problem feel free to re-open this issue. When you do re-open it, please include a specific command to reproduce the problem so that we can try to figure out what the problem is.
